nrf_rpc/CHANGELOG.rst

```diff
@@ -0,0 +1,106 @@
+.. _nrf_rpc_changelog:
+
+Changelog
+#########
+
+.. contents::
+   :local:
+   :depth: 2
+
+All notable changes to this project in the |NCS| are documented in this file.
+
+Main branch
+***********
+
+All the notable changes added to the |NCS| main are documented in this section.
+
+Added
+=====
+
+* Added configurations number of ZCBOR backups.
```

Comment:
A bit unclear what is "configurations number", needs to be clarified. ZCBOR -> zcbor.

Reply:
ACK. Changed to: _Added :kconfig:option:CONFIG_NRF_RPC_ZCBOR_BACKUPS to configure number of zcbor backups._